legacy/classes/cs_discussion_manager.php

```diff
@@ -365,7 +365,7 @@ function _performQuery($mode = 'select') {
 
      if ( isset($this->_sort_order) ) {
         if ( $this->_sort_order == 'latest' ) {
-           $query .= ' ORDER BY '.$this->addDatabasePrefix('discussions').'.latest_article_modification_date DESC, ' . $this->addDatabasePrefix('discussions') . '.modification_date DESC';
+           $query .= ' ORDER BY '.$this->addDatabasePrefix('discussions') . '.modification_date DESC';
```

Comment:
Is this really needed to reverse the order of the discussion articles? I think this will only affect the sorting of the discussion entries in the rubric's list.